src/core/config.ts

```diff
@@ -68,7 +68,6 @@ export const DEFAULT_CONFIG: ReactQueryConfig = {
 
 export function getDefaultReactQueryConfig() {
   return {
-    shared: { ...DEFAULT_CONFIG.shared },
```

Comment:
Is this a breaking change?

Reply:
The function returns the default config and shared is typed optional so this should not be breaking. Also this function has just been introduced in the previous release ;)